### PR TITLE
version 0.0.1 maybe?

### PR DESCRIPTION
**Pull request by Ivefan**

### CODE REVIEW

**Comment by Hanicef** [comment]
It's best to use tags for this.

File: README.md [hunk 1]
```diff
@@ -2,17 +2,16 @@
 This project was born on 31/10-18
 
 Five unemployed dudes with one given goal, one destination, to create a community for devoted video-game admirers to share their their thoughts and opinions.
-This quest meant a long distance from society and human interraction, but since none of them were familiar with it anyway they took their equipment and settled in a deep dark dungeon in the rainy forest of Scandanavia.
-<br />
+This quest meant a long distance from society and human interraction, but since none of them were familiar with it anyway they took their equipment and settled in a deep dark dungeon in the dank forest of Scandanavia.
 <br />
 
-######...legend says that they are stil in that dungeon
+###### ....legend says that they are stil in that dungeon
```
Comment (Hanicef) [inline README.md]: dats an important dot right there

File: src/main/java/com/communityratesgames/Application.java [hunk 2]
```diff
@@ -9,6 +9,6 @@
     public static void main(String[] args) {
 
         SpringApplication.run(Application.class, args);
-        System.out.println("Hallå eller gubbar!");
+        System.out.println("###Code Compiled Sucessfully###");
```
Comment (Hanicef) [inline src/main/java/com/communityratesgames/Application.java]: Is this neccesary? Spring boot already says OK if it has been deployed properly.

File: src/main/resources/application.properties [hunk 3]
```diff
@@ -10,8 +10,10 @@ spring.jpa.hibernate.ddl-auto=create-drop
 spring.jpa.show-sql=true
 spring.jpa.database=default
 spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.MySQL5InnoDBDialect
+spring.datasource.initialization-mode=always
 # server.address=172.18.128.205
```
Comment (Hanicef) [inline src/main/resources/application.properties]: Maybe remove this? I can't see a reason to use this in the future.

File: README.md (hunk 1 above)
Comment (ArastoSahbaei) [reply README.md]: roflmao

File: src/main/java/com/communityratesgames/Application.java (hunk 2 above)
Comment (ArastoSahbaei) [reply src/main/java/com/communityratesgames/Application.java]: feel free to remove diz

File: src/main/resources/application.properties (hunk 3 above)
Comment (ArastoSahbaei) [reply src/main/resources/application.properties]: should remain until we have a remote server?

Comment (Ivefan) [reply src/main/resources/application.properties]: Expand database population scrip with more data and move the trigger into a config file.
like data-init = true then use script

File: src/main/java/com/communityratesgames/Application.java [hunk 4]
```diff
@@ -9,6 +9,6 @@
...
     }
 }
```
Comment (Hanicef) [inline src/main/java/com/communityratesgames/Application.java]: I noticed that some of the files don't end with a newline, as seen here.
Just want to emphasize the importance of it; Git, as well as most other UNIX programs, expects files to end with a newline, and not doing so can confuse these programs. See https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline for more explanation.

Comment (Hanicef) [reply src/main/java/com/communityratesgames/Application.java]: I've fixed this now, but please think about this in the future.

**Pull request merged**